packages/slice-machine/server/src/api/services/getEnv.ts

```diff
@@ -1,5 +1,123 @@
-import getEnv from "../../../../lib/env/getEnv";
+import {
+  fromUrl,
+  parseDomain,
+  ParseResult,
+  ParseResultType,
+} from "parse-domain";
 
-export default async function handler() {
-  return getEnv();
+import { Models } from "@slicemachine/core";
```

Comment:
I'm guessing it must've been a pain to move it back here but thank you.
I really feel it does belong in the server.